evo/tools/tf_cache.py

```diff
@@ -172,10 +161,19 @@ def get_trajectory(self, reader: Rosbag1Reader,
             raise TfCacheException("Could not load trajectory: " + str(e))
         # rosbags Reader start_time is in nanoseconds.
         start_time = rospy.Time.from_sec(reader.start_time * 1e-9)
-        return self.lookup_trajectory(parent, child, start_time=start_time,
-                                      end_time=latest_time)
+        if timestamps is None:
+            timestamps = []
+            step = rospy.Duration.from_sec(1. /
+                                           SETTINGS.tf_cache_lookup_frequency)
+            time = start_time
+            while time < latest_time:
```

Comment:
here I should change this to
`while time <= latest_time` to support scenarios where the start_time==latest_time.

Reply:
should I add here the logic for the static tf?
```
if end_time.to_sec() == 0:
   timestamp = [end_time]
else:
   .....
```